### PR TITLE
Add Users Cypress Suite for React Example App

### PR DESCRIPTION
**Pull request by martinmckenna**
## JIRA Ticket(s) if any

* [MOB-3930](https://iterable.atlassian.net/browse/MOB-3930)

## Description

Adds cypress suite for Users page in React example app

## Test Steps

1. Run `yarn install:all && cd ./react-example && yarn cypress`
2. Run entire cypress suite in the pop-up window
3. Ensure all test pass

### CODE REVIEW

File: react-example/cypress/integration/users/users.spec.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    cy.get('[data-qa-update-user-email-input]').type('SomeItem');
+    cy.get('[data-qa-update-user-email-submit]').submit();
+
+    cy.wait('@updateUserEmail');
+
+    cy.get('[data-qa-update-user-email-response]').contains(
+      JSON.stringify({
+        msg: 'No API key found on request',
+        code: 'BadApiKey',
+        params: { ip: '68.111.205.112', endpoint: '/api/users/update' }
```
Comment (pauljung14) [inline react-example/cypress/integration/users/users.spec.ts:99]: just double checking, is this hardcoded IP address meant to be here?

File: react-example/cypress/integration/users/users.spec.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    cy.get('[data-qa-update-subscriptions-input]').type('10');
+    cy.get('[data-qa-update-subscriptions-submit]').submit();
+
+    cy.wait('@updateSubscriptions');
+
+    cy.get('[data-qa-update-subscriptions-response]').contains(
+      JSON.stringify({
+        msg: 'No API key found on request',
+        code: 'BadApiKey',
+        params: { ip: '68.111.205.112', endpoint: '/api/users/update' }
```
Comment (pauljung14) [inline react-example/cypress/integration/users/users.spec.ts:151]: same as above

**Review by pauljung14 — APPROVED** [review]
one comment

File: react-example/cypress/integration/users/users.spec.ts (hunk 1 above)
Comment (martinmckenna) [reply react-example/cypress/integration/users/users.spec.ts:99]: it is, but good call! Let me make the error more generic

**Pull request merged**